src/test/java/io/openmanufacturing/sds/aspectmetamodel/AbstractShapeTest.java

```diff
@@ -73,71 +73,47 @@ protected static Stream<KnownVersion> versionsUpToIncluding1_0_0() {
    final String MESSAGE_LANG_NOT_UNIQUE = "Language \\\"en\\\" used more than once";
    final String MESSAGE_INVALID_LANG_STRING = "Value must be a valid literal of type langString";
    final String MESSAGE_DUPLICATE_PROPERTY = "Property may only have 1 value, but found 2";
-   final String MESSAGE_MISSING_DATATYPE = "The datatype is not defined on the Characteristic instance (see focusNode).";
-   final String MESSAGE_INVALID_ENTRY_ASPECT_PROPERTY_LIST = "Each element in the Aspect's properties list must be either a Property or a blank node "
-         + "referring to a Property and defining bamm:optional \\\"true\\\"^^xsd:boolean and/or bamm:payloadName.";
-   final String MESSAGE_INVALID_ENTRY_ENTITY_PROPERTY_LIST = "Each element in the Entity's properties list must be either a Property or a blank node "
-         + "referring to a Property and defining either bamm:optional \\\"true\\\"^^xsd:boolean and/or bamm:payloadName or bamm:notInPayload "
-         + "\\\"true\\\"^^xsd:boolean.";
-   final String MESSAGE_INVALID_ENTITY_PROPERTY_DEFINITION = "A Property of an Entity may not be defined with both bamm:optional \\\"true\\\"^^xsd:boolean "
-         + "and bamm:notInPayload \\\"true\\\"^^xsd:boolean.";
-   final String MESSAGE_INVALID_ENTITY_PROPERTY_PAYLOAD_NAME = "A Property of an Entity may not be defined with both bamm:payloadName and bamm:notInPayload "
-         + "\\\"true\\\"^^xsd:boolean.";
-   final String MESSAGE_ENTITY_NOT_USED_IN_ENUMERATION = "A Property of an Entity may only be defined with bamm:notInPayload \\\"true\\\"^^xsd:boolean when "
-         + "the Entity, or one of its parent Entities, is used as the data type of an Enumeration.";
-   final String MESSAGE_INVALID_ENTITY_WITH_SINGLE_PROPERTY = "An Entity defining a Property with bamm:notInPayload \\\"true\\\"^^xsd:boolean must define at "
-         + "least one more Property.";
-   final String MESSAGE_INVALID_QUANTITY_KIND = "Unit of Duration must have quantity kind time";
-   final String MESSAGE_VALUE_NOT_OF_DATATYPE = "One of the values is not of the specified data type.";
+   final String MESSAGE_MISSING_DATATYPE = "No datatype is defined on the Characteristic instance '{$this}'.";
+   final String MESSAGE_INVALID_ENTRY_ENTITY_PROPERTY_LIST =
+         "Element '{?value}' in the Entity's '{$this}' properties list must be either a Property or a blank node "
```

Comment:
"Element '{?value}' in the Entity's '{$this}' properties list must be either a Property or a blank node."